crates/indexer/src/models/processor_statuses.rs

```diff
@@ -48,9 +46,9 @@ impl ProcessorStatus {
         success: bool,
         details: Option<String>,
     ) -> Vec<Self> {
-        let mut status: Vec<Self> = vec![Self::new(name, start_version, success, details.clone())];
-        for version in start_version + 1..end_version {
-            status.push(Self::new(name, version, success, details.clone()));
+        let mut status: Vec<Self> = vec![];
+        for version in start_version..(end_version + 1) {
```

Comment:
why the switch? Isnt it possible end version doesn't exist yet?

Reply:
This was missing the end version. This works for Genesis block